README.md

```diff
@@ -1,113 +1,96 @@
-<p align="center"><img src="https://github.com/jetstack/cert-manager/blob/master/logo/logo.png" width="250x" /></p>
-</a>
-<a href="https://godoc.org/github.com/cert-manager/istio-csr"><img src="https://godoc.org/github.com/cert-manager/istio-csr?status.svg"></a>
-<a href="https://goreportcard.com/report/github.com/cert-manager/istio-csr"><img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/cert-manager/istio-csr" /></a></p>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/jetstack/cert-manager/master/logo/logo.png" width="250x" alt="cert-manager project logo" />
+</p>
+<p align="center">
+  <a href="https://godoc.org/github.com/cert-manager/istio-csr">
+    <img src="https://godoc.org/github.com/cert-manager/istio-csr?status.svg">
+  </a>
+  <a href="https://goreportcard.com/report/github.com/cert-manager/istio-csr">
+    <img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/cert-manager/istio-csr" />
+  </a>
+  <a href="https://artifacthub.io/packages/search?repo=cert-manager">
+    <img alt="artifact hub badge" src="https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/cert-manager">
+  </a>
+</p>
 
 # istio-csr
 
 istio-csr is an agent that allows for [Istio](https://istio.io) workload and
 control plane components to be secured using
-[cert-manager](https://cert-manager.io). Certificates facilitating mTLS, inter
-and intra cluster, will be signed, delivered and renewed using [cert-manager
-issuers](https://cert-manager.io/docs/concepts/issuer).
+[cert-manager](https://cert-manager.io).
 
-⚠️ Currently supports Istio versions v1.7+
+Certificates facilitating mTLS &mdash; both inter
+and intra-cluster &mdash; will be signed, delivered and renewed using [cert-manager
+issuers](https://cert-manager.io/docs/concepts/issuer).
 
-⚠️ Currently supports cert-manager versions v1.3+
+istio-csr supports Istio v1.7+ and cert-manager v1.3+
 
 ---
 
-## Installation
+## Getting Started Guide For istio-csr
+
+We have [a guide](./docs/getting_started.md) for setting up istio-csr in a fresh [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) cluster.
+
+Following the guide is the best way to see istio-csr in action.
 
-### Installing cert-manager
+If you've already seen istio-csr in action or if you're experienced with running Istio and just want quick installation instructions, read on for more details.
 
-Firstly, [cert-manager must be
-installed](https://cert-manager.io/docs/installation/) in your cluster.
+## Lower-Level Details (For Experienced Istio Users)
 
-### Issuer or ClusterIssuer
-An Issuer or ClusterIssuer must be configured which will be used to sign Istio
-certificates against.
+⚠️  The [getting started](./docs/getting_started.md) guide is a better place if you just want to try istio-csr out!
```

Comment:
Looks like this is repeated from 8 lines up.

Reply:
This was intentional, I wanted it to be _super_ clear that the README isn't the place for newbies